api/server/services/AuthService.js

```
@@ -103,31 +115,46 @@ const sendVerificationEmail = async (user) => {
+
+  if (!user) {
+    logger.warn(`[verifyEmail] [User not found] [Email: ${decodedEmail}]`);
+    return new Error('User not found');
+  }
+
+  if (user.emailVerified) {
+    logger.info(`[verifyEmail] Email already verified [Email: ${decodedEmail}]`);
+    return { message: 'Email already verified', status: 'success' };
+  }
+
+  let emailVerificationData = await findToken({ email: decodedEmail });
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'emailVerificationData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)